src/css/home.scss

```diff
@@ -209,22 +209,29 @@
 
             li {
                 list-style: none;
-                margin: 1em 0;
+                margin: 2.14rem 0;
+                text-align: center;
 
                 a {
                     display: inline-block;
-                    @include hide-button-text;
 
-                    &.compound {
-                        width: 181px;
-                        height: 41px;
-                        background-image: url(../components/svgs/compound-logo.svg);
+                    .gatsby-image-wrapper {
+                        [aria-hidden="true"] {
+                            display: none;
+                        }
+
+                        img {
+                            position: static !important;
+                            height: auto !important;
+                            width: auto !important;
```

Comment:
How come we have to `!important` here?

Reply:
I needed to override the inline styles set by the gatsby-image component. In retrospect, I should test out if using a [`fixed` query](https://www.gatsbyjs.org/docs/gatsby-image/#images-with-a-fixed-width-and-height) instead of a `fluid` one here would allow us to avoid this situation...I assumed we needed fluid but it wasn't working in the way I wanted to.